src/theme/Layout/index.tsx

```diff
@@ -34,8 +38,34 @@ const DEFAULT_API_BASE_DEV_URL = 'https://auth.logto.dev';
 
 export default function LayoutWrapper(props: Props): ReactNode {
   const [config, setConfig] = useState<GoogleOneTapConfig | undefined>(undefined);
+  const [authStatus, setAuthStatus] = useState<boolean | undefined>(undefined);
+  const [authCheckError, setAuthCheckError] = useState<string | undefined>(undefined);
```

Comment:
nit: Can be simplified to:
```suggestion
  const [authStatus, setAuthStatus] = useState<boolean>();
  const [authCheckError, setAuthCheckError] = useState<string>();
```